scripts/repeatIT.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+targetTest=${1:-ExpositionIT}
+
+logFile=/tmp/mvn-${targetTest}-$(date +"%d-%m-%Y_%H:%M").log
+
+mvn clean install -DskipTests=true
+
+for COUNT in {1..10} ; do
+    echo "=============== RUN ${COUNT} STARTING =============== " | tee --append "${logFile}"
+    mvn verify -Dit.test=${targetTest} -pl :integrationtests >> "${logFile}"
+    EC=$?
+    echo "=============== RUN ${COUNT} FINISHED =============== " | tee --append "${logFile}"
```

Comment:
We're using `tee --append` here too

Reply:
fixed in b6d9136c3eee0ef32cbf7b6916cc6877b5c7d71b